dts/examples/stm32f429i_disc1_autotest.dts

```diff
@@ -66,6 +66,19 @@
 	};
 };
 
+&timers5 {
```

Comment:
TIM5 seems to have the position 50 (not 49) in STM32F42xxx reference manual (it is 49 for STM32U5).

Reply:
yep, defined in official dtsi, not here :)